src/js/thirdparty/express-bun.ts

```
@@ -0,0 +1,695 @@
+  get headersSent(): boolean {
+    return this._sent;
+  }
+
+  get locals(): Record<string, any> {
+    return this._locals;
+  }
+
+  status(code: number): this {
+    this._statusCode = code;
+    return this;
+  }
```

Biome (2.1.2)

[error] Duplicate class member name "status"



(lint/suspicious/noDuplicateClassMembers)